packages/payment-processor/src/payment/index.ts

```diff
@@ -27,6 +27,7 @@ export const supportedNetworks = [
   ExtensionTypes.ID.PAYMENT_NETWORK_ERC20_PROXY_CONTRACT,
   ExtensionTypes.ID.PAYMENT_NETWORK_ERC20_FEE_PROXY_CONTRACT,
   ExtensionTypes.ID.PAYMENT_NETWORK_ETH_INPUT_DATA,
+  ExtensionTypes.ID.PAYMENT_NETWORK_NATIVE_TOKEN,
```

Comment:
Should we remove `PAYMENT_NETWORK_ETH_INPUT_DATA` ?
```suggestion
  ExtensionTypes.ID.PAYMENT_NETWORK_NATIVE_TOKEN,
```

Reply:
Invoicing use also ETH_INPUT_DATA, it might require more refacto, no?
(in payment.tsx)